src/screens/Onboarding/StepAlgoFeeds/index.tsx

```diff
@@ -20,7 +20,7 @@ import {
   OnboardingControls,
 } from '#/screens/Onboarding/Layout'
 import {FeedCard} from '#/screens/Onboarding/StepAlgoFeeds/FeedCard'
-import {IconCircle} from '#/screens/Onboarding/IconCircle'
+import {IconCircle} from '#/components/IconCircle'
```

Comment:
I think we might find more use for this icon circle throughout the app as we implement ALF so moving it to the root components directory.